compiler/src/codegen/compcore.re

```diff
@@ -500,393 +301,80 @@ let call_malloc = (wasm_mod, env, args) => {
   let args = [
     Expression.global_get(
       wasm_mod,
-      get_imported_name(malloc_mod, malloc_closure_ident),
+      get_imported_name(gc_mod, malloc_closure_ident),
       Type.int32,
     ),
     ...args,
   ];
   Expression.call(
     wasm_mod,
-    get_imported_name(malloc_mod, malloc_ident),
+    get_imported_name(gc_mod, malloc_ident),
     args,
     Type.int32,
   );
 };
-let call_incref = (wasm_mod, env, arg) =>
-  if (Config.no_gc^) {
-    arg;
-  } else {
-    Expression.call(
-      wasm_mod,
-      get_imported_name(runtime_mod, incref_ident),
-      [arg],
-      Type.int32,
-    );
-  };
-let call_incref_adt = (wasm_mod, env, arg) =>
```

Comment:
I perhaps would have left these variant functions as aliases to `call_incref`, since we could inject debugging information in the future.